Feature/static content

jagannath-gupta opened the pull request:
Added Static pages for Top Navigation Menu items with dummy content. Kindly review the pages and Navigation Menu items on Header.

ajayvfox reviewed (commented):
Thanks for creating a separate feature branch! This is SO much easier for me to review. 

Few points:

1) Links on the Home page look good.
2) The same links will show on the 'Dashboard' after user logs in, correct? Can you please add those?
3) You don't need to create PR until you're ready to merge. You can just give me the branch name - if you would like - OR - you can create a PR. It's up to you.

ajayvfox committed:
Merge branches 'develop' and 'feature/Static_Content' of https://github.com/vfoexchange/vfox0.1 into feature/Static_Content

ajayvfox commented on vfox-web/src/app/views/login/login.component.html:
```diff
@@ -1,17 +1,24 @@
             <div class="home_heading">Welcome to VFOX<br/> (Virtual Family Office Exchange)</div>
             <div class="clearfix"></div>
             <p class="home_txt">Preservation and Enhancement of Wealth</p>
+            <p class="home_txt">The Disruptive Wealth Management Platform for Private Wealth<br/>(Individuals, Families & Corporations)</p>
```
Can we change this "Disruptive..." to a smaller font.

ajayvfox commented on vfox-web/src/app/views/static-page/about.component.html:
```diff
@@ -0,0 +1,30 @@
+<div class="container">
```
'About Us' looks great to me. Few more suggestions:
1) Clicking on 'Dashboard' could go to 'Select Services' for now. We don't have contents for a 'blank' page.
2) Shouldn't we have a 'Home' page? There's no way to go to home unless I 'SignOut'. If user is already signed in show the 'Dashboard' link; otherwise don't show it.
3) Logos seem to have disappeared from the pages such as 'Tax Mitigation' etc. Let me see if this is a problem on my end.

jagannath-gupta committed:
[Day commit] : Updated UI

jagannath-gupta commented:
Kindly review the 'About Us' page and added navigation menu items after User login. We have also added Captcha on Advisor Registration page for review.

ajayvfox committed:
Merge branch 'feature/Static_Content' of https://github.com/vfoexchange/vfox0.1 into feature/Static_Content

jagannath-gupta committed:
[vfox-web]: Font-size updates

jagannath-gupta committed:
[vfox-web]: Updated Dashboard link to Select Services page

jagannath-gupta committed:
[vfox-web]: Removed package-lock.json

jagannath-gupta committed:
[vfox-web]: Static page - UI integration for Contact page

jagannath-gupta commented:
Kindly review the Contact Us Static page UI.

ajayvfox committed:
Merge branch 'feature/Static_Content' of https://github.com/vfoexchange/vfox0.1 into feature/Static_Content

ajayvfox committed:
Added content to 'What we do' page.
Fixed Map & Phone number in 'Contact Us' page.

ajayvfox committed:
Added a 'generic' introduction for all service providers.

jagannath-gupta committed:
[vfox-web]: Updated UI for What we do page and added link on header for Services.

jagannath-gupta committed:
[vfox-web]: UI updated for Top menu item for services drop-down

jagannath-gupta committed:
[vfox-web]: Updated CSS for Dropdown menu items

jagannath-gupta committed:
[vfox-web]: Updated CSS for dropdown hover link

jagannath-gupta committed:
[vfox-web]: updated UI for drop-down-menu items

jagannath-gupta committed:
[vfox-web]: Updated UI for whatwedo static content page

pradeepvfox committed:
[vfox-web] - Modified header and add services template for static pages

jagannath-gupta committed:
[Day commit]: vfox-web - UI integration of Services page for Tax Mitigation, Insurance and Alternate Investment

pradeepvfox committed:
[vfox-web] - contact us page API Integration

pradeepvfox committed:
[Day commit] - resolved aot build issue

jagannath-gupta committed:
[vfox-web]:static-page - error message text color css updated in contact form page

jagannath-gupta committed:
[vfox-web]: Updated css

jagannath-gupta committed:
[vfox-web]: UI updates on Footer page

pradeepvfox committed:
[Day commit] - Add footer in all static pages

pradeepvfox committed:
[Day commit] - Add footer file

jagannath-gupta committed:
[vfox-web]: Footer UI updated

pradeepvfox committed:
[vfox-web] - disable maintain website link

pradeepvfox commented:
Kindly review the 'Contact Us' page form submission and added footer in all static pages.

pushpendranirwan committed:
Merge branch 'develop' into feature/Static_Content

pradeepvfox committed:
[Day commit] - Reolved conflict from home component

ajayvfox committed:
Added properties related to SSL. All of them are commented out.

ajayvfox reviewed (approved):
LGTM

jagannath-gupta commented:
Kindly review :
a) Submenus popover into 'Maintain website' link
b) Coming Soon page UI integrated and linked with sub-menus
c) All services added from 'Insurance',.... 'Business Services' into 'Services' drop-down list
